src/graphql/admin/index.ts

```diff
@@ -13,7 +13,7 @@ export const gqlAdminSchema = new GraphQLSchema({
   types: ALL_INTERFACE_TYPES,
 })
 
-if (!isProd) {
+if (!isProd && !isTest) {
```

Comment:
Just for my own knowledge, how does skipping the Prettier formatting for tests help with the open handles problem?

Reply:
not sure if I understand but... this was creating random open handles because write of `schema.graphql` file is async and sometimes tests finished before that.